Makefile

```diff
@@ -16,9 +20,8 @@ export CGO_ENABLED=0
 
 # Environmental details
 BINDIR := rootfs/usr/local/bin
-VERSION := 0.0.1-$(shell date "+%Y%m%d%H%M%S")
-LDFLAGS := "-s -X main.version=${VERSION}"
-IMAGE := ${DEIS_REGISTRY}/deis/${SHORT_NAME}:${VERSION}
+LDFLAGS := "-s -X main.version=${BUILD_TAG}"
+IMAGE := ${DEIS_REGISTRY}/deis/${SHORT_NAME}:${BUILD_TAG}
```

Comment:
Right, the hacky workaround that exists elsewehere is to set `DEIS_REGISTRY=deisci/` with a trailing slash and remove that slash from the `IMAGE` line. Then doing `DEIS_REGISTRY='' make docker-build docker-push` works correctly for Docker Hub, which won't accept `docker.io` or any other prefix for `docker push`.